compute/cloud-client/src/test/java/compute/disks/HyperdisksIT.java

```diff
@@ -113,10 +113,11 @@ public void stage1_CreateHyperdiskStoragePoolTest()
     Assert.assertEquals(10240, storagePool.getPoolProvisionedCapacityGb());
     Assert.assertTrue(storagePool.getStoragePoolType().contains("hyperdisk-balanced"));
     Assert.assertTrue(storagePool.getCapacityProvisioningType().equalsIgnoreCase("advanced"));
+    Assert.assertTrue(storagePool.getPerformanceProvisioningType()
+        .equalsIgnoreCase(PERFORMANCE_PROVISIONING_TYPE));
     Assert.assertTrue(storagePool.getZone().contains(ZONE));
   }
 
-  @Disabled
   @Test
   public void stage2_CreateHyperdiskStoragePoolTest()
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
```

Comment:
what is being testing in this method that was not tested in the previous method? Why changing the arguments to the call to `CreateDiskInStoragePool.createDiskInStoragePool()` should matter for testing this code sample?
The code sample method does not have any conditions, so testing Google API invocation with different parameters does not test code sample but the backend API which is unnecessary.

Reply:
I've fixed naming for these tests. As I understand from code and tests 
1. the first testCreateHyperdiskStoragePool() tests that StoragePoll was created as expected
2. the second testCreateDiskInStoragePool() takes the StoragePoolLink of the StoragePool created in the first test and tests it's parameters.

Reply:
according to your explanations the second test does not validate the code sample. the code sample shows how to create a storage pool. the test(s) should validate the code.

please, modify the test class to do one of the following, either to remove the second test because it does not add to the validation to code sample or perform all validations / asserts in the first test method.

Reply:
We have two classes - CreateDiskInStoragePool and CreateHyperdiskStoragePool. To test this code Svyatoslav created two test methods - testCreateDiskInStoragePool and testCreateHyperdiskStoragePool. As far as I can see, these tests are working properly